packages/cli/src/lib/bundler/bundle.ts

```diff
@@ -90,10 +89,10 @@ export async function buildBundle(options: BuildOptions) {
   );
 }
 
-async function build(compiler: webpack.Compiler, isCi: boolean) {
+async function build(config: webpack.Configuration, isCi: boolean) {
   const stats = await new Promise<webpack.Stats | undefined>(
     (resolve, reject) => {
-      compiler.run((err, buildStats) => {
+      webpack(config, (err, buildStats) => {
```

Comment:
Just currious: Why don't we need to explicitly call `.run` here? Is the callback in the second parameter for `webpack(...)` doing the same as explicitly calling run?

Reply:
Thanks!